tests/instrumentation-framework/src/test/resources/testTicketMonsterDockerCompose/docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright 2015-2016 Red Hat, Inc. and/or its affiliates
+# and other contributors as indicated by the @author tags.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+version: "2"
+
+services:
+  ticket-monster:
+    image: pavolloffay/jboss-ticket-monster
+    ports:
+      - "8080:8080"
```

Comment:
For tests, I'd try not to bind to 8080 on localhost, as changes are high that something is running there. I'd use 10080, or some other high port.